<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formHypercubeAnalysis</class>
 <widget class="QDialog" name="formHypercubeAnalysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hypercube Analysis</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QSlider" name="slideTmpImg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>48</y>
     <width>811</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelImgNumber">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCubeToImg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbLoadCube">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Cube</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>251</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelProgressBar">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>38</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGraphicsView" name="gvImg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="gvPlot">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
